src/Command.js

```diff
@@ -14,6 +14,7 @@ export default class Command {
     this.repository = new Repository();
     this.progressBar = progressBar;
     this.logger = logger;
+    this.concurrency = (!flags || flags.concurrency === undefined) ? 4 : flags.concurrency;
```

Comment:
It was broken in the tests actually – `flags` was null. I imagine that'd also be the case if you used commands programatically.

I also check for undefined specifically so that it's not `0 || 4`, so that if you specify `0` you get an error, rather than it just using `4`.